CHANGES.rst

```diff
@@ -22,6 +22,11 @@
     subcommand [GH-1079]
   - Add ``:enabled`` property to ``flycheck-define-checker`` [GH-1089]
 
+- Improvements:
+
+  - When checking Javascript buffer we will disable ``eslint`` checker if that
+    checker can not find any valid ESLint configuration [GH-1085]
```

Comment:
I think we can shorten that to "Do not use \`\`javascript-eslint\`\` if eslint cannot find a valid configuration [GH-1085]".